src/workloads/execution/BackgroundIndexConstruction.yml

```diff
@@ -0,0 +1,156 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload tests the write performance impact of creating indexes on a very large collection.
+  It synthetically lowers the number of concurrent operations allowed to represent a saturated
+  server.
+
+Keywords:
+- stress
+- indexes
+- InsertRemove
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 1024
+      socketTimeoutMS: 3600000  # = 1 hour
+
+Actors:
+# Phase 1: Insert enough data to ensure than an index build on each field spills to disk with a
+# memory limit of 100MB.
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000000
+    BatchSize: 1000
+    Document:
+      randomInt: {^RandomInt: {min: -10000000, max: 10000000}}
+      randomString: {^RandomString: {length: 16}}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: Lower the memory limit for spilling to disk so that it occurs more often.
+- Name: Setup
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operations:
+    # Fsync to force a checkpoint and quiesce the system.
+    - OperationMetricsName: FsyncCommand
+      OperationName: AdminCommand
+      OperationCommand:
+        fsync: 1
+    - OperationName: AdminCommand
+      OperationCommand:
+        setParameter: 1
+        storageEngineConcurrentWriteTransactions: 5
+    - OperationMetricsName: LimitIndexBuildMemoryUsageCommand
+      OperationName: RunCommand
+      OperationCommand:
+        setParameter: 1
+        maxIndexBuildMemoryUsageMegabytes: 100
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phases 3+: Build indexes on each field.
+- Name: IndexCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  # Build an index on an integer field.
+  - Repeat: 1
+    Database: *db
+    Operations:
+    - OperationMetricsName: CreateIndexInt
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: &coll Collection0
+        indexes:
+        - key:
+            randomInt: 1
+          name: random_int
+    - OperationMetricsName: DropIndexInt
```

Comment:
This is interesting. I frankly don't remember how index builds work these days. Is there a chance of the background workload speeding up as more of the index is completed, and then falling back down with the index is dropped? 

I suspect there are phases to the index build, and their impact (each phase) on the background workload may be different. It's worth looking at FTDC data from your runs to see if there's any interesting periodicity with the index builds.